test/e2e/e2e.go

```diff
@@ -172,4 +174,105 @@ var _ = ginkgo.Describe("[efs-csi] EFS CSI", func() {
 	ginkgo.Context(testsuites.GetDriverNameWithFeatureTags(driver), func() {
 		testsuites.DefineTestSuite(driver, csiTestSuites)
 	})
+
+	f := framework.NewDefaultFramework("efs")
+
+	ginkgo.Context(testsuites.GetDriverNameWithFeatureTags(driver), func() {
+		ginkgo.It("should mount different paths on same volume on same node", func() {
+			ginkgo.By(fmt.Sprintf("Creating efs pvc & pv with no subpath"))
+			pvcRoot, pvRoot, err := createEFSPVCPV(f.ClientSet, f.Namespace.Name, f.Namespace.Name+"-root", "/")
+			framework.ExpectNoError(err, "creating efs pvc & pv with no subpath")
+			defer func() { _ = f.ClientSet.CoreV1().PersistentVolumes().Delete(pvRoot.Name, &metav1.DeleteOptions{}) }()
+
+			ginkgo.By(fmt.Sprintf("Creating pod to make subpaths /a and /b"))
+			pod := e2epod.MakePod(f.Namespace.Name, nil, []*v1.PersistentVolumeClaim{pvcRoot}, false, "mkdir /mnt/volume1/a && mkdir /mnt/volume1/b")
+			pod, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(pod)
+			framework.ExpectNoError(err, "creating pod")
+			framework.ExpectNoError(e2epod.WaitForPodSuccessInNamespace(f.ClientSet, pod.Name, f.Namespace.Name), "waiting for pod success")
+
+			ginkgo.By(fmt.Sprintf("Creating efs pvc & pv with subpath /a"))
+			pvcA, pvA, err := createEFSPVCPV(f.ClientSet, f.Namespace.Name, f.Namespace.Name+"-a", "/a")
```

Comment:
Wish Golang support keywords argument like Python does. It's not easy to tell what those parameters really represent.

Reply:
agree lol. a pattern I have seen is to add /* <parameter name> */ after every arg but thats gets tiring and verbose fast. IDE can help here.